person.rb

```diff
@@ -0,0 +1,22 @@
+class Person
+  def initialize(age, name = 'Unknown', parent_permission: true)
+    @id = Random.rand(1..1000)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  attr_reader :id
+
+  attr_accessor :age, :name
```

Comment:
- Hey, for readability and following the convention used in ruby, kindly move these line up  before the constructor.